tests/security/cc/kvm_svirt_apparmor.pm

```diff
@@ -0,0 +1,48 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Run 'kvm_svirt_apparmor' test case of 'audit-test' test suite
+# Maintainer: Liu Xiaojing <xiaojing.liu@suse.com>
+# Tags: poo#101761
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use Mojo::File 'path';
+use audit_test 'prepare_for_test';
+
+sub run {
+    my ($self) = shift;
+
+    select_console 'root-console';
+
+    # the steps of testing kvm_svirt_apparmor is not same as other audit-test,
+    # so we need to do the `make` in the test case directory and run the test.
+    prepare_for_test();
+
+    assert_script_run('cd kvm_svirt_apparmor/');
+    assert_script_run('make');
+    assert_script_run('cd tests/');
+    assert_script_run('./vm-sep');
+
+    # We check the test result by parse log file in /tmp/vm-sep/vm-sep-crack.log
+    my $log_file = '/tmp/vm-sep/vm-sep-crack.log';
+    assert_script_run("[[ -e $log_file ]]");
+    my $test_results = {};
+    my $output = script_output("cat $log_file");
+    my @lines = split(/\n/, $output);
+    foreach (@lines) {
```

Comment:
Xiaojing, as the base line has passed all cases, right? so can you please add some more comments here?

My concern is, if the base line is failed, and the new run is passed, we should mark it pass rather than fail

Reply:
There is no base line here. The file we parsed here is generated by the test. Its content looks like:
```
Label setup for cracker successful - PASSED
TEST CASE: sending signals
Sending signal 0 to slave 12007: denied - PASSED
... ...
TEST CASE: accessing disk
Test existance of file /tmp/vm-sep/vm-sep-slave.disk: PASSED
Test read access to file /tmp/vm-sep/vm-sep-slave.disk prevented: (stat/access allowed) PASSED
Test write access to file /tmp/vm-sep/vm-sep-slave.disk prevented: (stat/access allowed) PASSED
Test execute access to file /tmp/vm-sep/vm-sep-slave.disk prevented: PASSED
TEST CASE: testing ptrace protection
/usr/local/eal4_testing/audit-test/kvm_svirt_apparmor/tests/ptrace-test 12007
current pid: 
ptrace prevented with EPERM - test PASSED
Expected error of EPERM received
TEST CASE: confining children
Children of confined process still confined: PASSED
TEST CASE: access restriction enforcement for USB device files
Write access to USB device assigned to other VM prevented: PASSED
=============================
Number of tests executed: 39
Number of tests failed:   0
Number of tests passed:   39

```
So I check the test result by checking if the total tests number is equal to the passed number.

Reply:
Got it.